appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -478,7 +484,9 @@ public static Link getBuddyUserLink(String rel, UUID userId, UUID requestingUser
 		private final String deviceFirebaseInstanceId;
 
 		@JsonCreator
-		public PostPutUserDto(@JsonProperty("firstName") String firstName, @JsonProperty("lastName") String lastName,
+		public PostPutUserDto(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 12 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS00107)